build/build.xml

```diff
@@ -101,7 +101,7 @@
 
         <move file="../temp/${name}.ane" todir="../bin"/>
         <!--<delete dir="../temp"/>-->
-        <copy file="../bin/${name}.ane" todir="${ane.folder}" overwrite="true"/>
+        <!-- <copy file="../bin/${name}.ane" todir="${ane.folder}" overwrite="true"/> -->
```

Comment:
don't know wtf this was intended to do, but it was creating a directory called ${ane.folder}, which was not making the world a safer or better place.